### DISCUSSION TITLE
Clean up Stale Branches for simplicity

### DISCUSSION THREAD

**Discussion by nerrixde** [ory/kratos#1219] [Ideas]
... and the "Forums" link in readme is outdated

**Comment by vinckr** [comment]
Thanks a lot for heads up ! 
I updated the README to point to github discussions instead of the old forum. 
As for the stale branches I think @zepatrik has to look at that.

**Reply by zepatrik** [reply]
Cleaned up all that had a closed PR or came from automation. :heavy_check_mark:
